ui/tx/TxDetails.tsx

```diff
@@ -431,7 +431,8 @@ const TxDetails = () => {
         ) }
         { data.tx_burnt_fee && !config.UI.views.tx.hiddenFields?.burnt_fees && !config.features.optimisticRollup.isEnabled && (
           <DetailsInfoItem
-            title="Burnt fees"
+            title="Reward pool fees"
+            // DOUBLE CHECK: is this calculation correct with the new naming?
```

Comment:
`ZEN awarded to the Reward Pool from transactions included in the block (Base fee (per unit of gas) * Gas Used).`